overlord/snapstate/backend/copydata_test.go

```diff
@@ -582,6 +583,114 @@ func (s *copydataSuite) TestUndoCopyDataSameRevision(c *C) {
 	}
 }
 
+const (
+	mountRunMntUbuntuSaveFmt = `26 27 8:3 / %s/run/mnt/ubuntu-save rw,relatime shared:7 - ext4 /dev/fakedevice0p1 rw,data=ordered`
+	mountSnapSaveFmt         = `26 27 8:3 / %s/var/lib/snapd/save rw,relatime shared:7 - ext4 /dev/fakedevice0p1 rw,data=ordered`
+)
+
+func mockUbuntuCore() (restore func()) {
+	onClassic := release.OnClassic
+	release.OnClassic = false
+	return func() { release.OnClassic = onClassic }
+}
```

Comment:
there is release.MockOnClassic that does this?

Reply:
I can't explain this one, fixed!